packages/suite/src/components/firmware/ReconnectDevicePrompt.tsx

```diff
@@ -255,6 +255,7 @@ export const ReconnectDevicePrompt = ({
 }: ReconnectDevicePromptProps) => {
     const { device } = useDevice();
     const { isWebUSB } = useFirmware();
+    console.log('render ReconnectDevicePrompt');
```

Comment:
although hook params in `useRebootRequest.ts` do not change, this hooks is called twice because this component rerenders.